jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -300,9 +301,14 @@ boolean getAllowInsecureRegistries() {
     return allowInsecureRegistries;
   }
 
+  /**
+   * Directory where non-java container files should be placed.
+   *
+   * @return path to extras directory, if not absolute then resolve against this project directory.
```

Comment:
wording change?
`path to the extra files directory. If the directory is not absolute, then it should have been resolved against the current project directory.`

Reply:
looks like I don't need this comment at all. I change this section of code.